.travis.yml

```diff
@@ -102,8 +101,8 @@ script:
         done;
       fi
     - cd .. && mv julia julia2
-    - cp /tmp/julia/lib/julia/sys.ji local.ji && /tmp/julia/bin/julia -J local.ji -e 'true' &&
-        /tmp/julia/bin/julia-debug -J local.ji -e 'true' && rm local.ji
+    - /tmp/julia/bin/julia --precompiled=no -e 'true' &&
+      /tmp/julia/bin/julia-debug --precompiled=no -e 'true'
```

Comment:
leave the indent otherwise it's not immediately obvious this is a line continuation